packages/node/cspell-ext.json

```diff
@@ -0,0 +1,42 @@
+// cSpell Settings
+{
+    "id": "node",
+    "name": "Node.js",
+    "description": "Node.js dictionary for cspell.",
+    // List of dictionary files to add to the global list of dictionaries
+    "dictionaryDefinitions": [
+        {
+            "name": "node",
+            "file": "./node.txt.gz",
+            "description": "Node.js dictionary for cspell."
+        }
+    ],
+    // Dictionaries to always be used.
+    // Generally left empty
+    "dictionaries": [],
+    // Language Rules to apply to matching files.
+    // Files are matched on `languageId` and `local`
+    "languageSettings": [
+        {
+            // VSCode languageId. i.e. typescript, java, go, cpp, javascript, markdown, latex
+            // * will match against any file type.
+            "languageId": "javascript",
```

Comment:
Note sure if there are some others that should be included or if this should just be a `*`